src/index.html

```diff
@@ -11,6 +11,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="form"
+      data-qa="big">
+      <input
+        class="form__input large"
+        type="text"
+        name="search"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      ></input>
```

Comment:
```suggestion
      <input
        class="form__input large"
        type="text"
        name="search"
        data-qa="keypress"
        placeholder="Try &quot;Los Angeles&quot;"
      />
```

Reply:
I don't understand this, as input is not self close tag, and when I tried to do this, code underlined red.
<img width="739" alt="Screenshot 2024-01-04 at 7 37 57 PM" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/151964804/72b341aa-3035-4c03-8236-905b84f45928">